src/anm/analytics.js

```diff
@@ -19,23 +19,17 @@ var Analytics = function () {
         if (utils.is.not_empty(self.queue)) {
             var array = [];
             for (var key in self.queue) {
-                array.push(self.queue[key]);
+                if (self.queue.hasOwnProperty(key)) {
```

Comment:
Since you use `forEach` (ES5+) above anyway, I think you also may use smth like 

```javascript
var array = Object.keys(self.queue).map(function (key) { return self.queue[key]; });
```